go.mod

```diff
@@ -167,6 +167,8 @@ require (
 	github.com/mailgun/minheap v0.0.0-20170619185613-3dbe6c6bf55f // indirect
 )
 
+require github.com/gravitational/teleport-plugins v0.0.0-20230210191358-1defd181af97
```

Comment:
can you move this block into any require block above?

Reply:
> Should we be pointing at a tag instead of a random commit?

This has the same problem as `gravitational/teleport/api`: we tag major versions, but do not change the import path as Go expects:

```console
$ go get github.com/gravitational/teleport-plugins@v12.0.2
go: github.com/gravitational/teleport-plugins@v12.0.2: invalid version: module contains a go.mod file, so module path must match major version ("github.com/gravitational/teleport-plugins/v12")
```